infrastructure/tooling/src/smoketest/checks/indexTask.js

```diff
@@ -0,0 +1,50 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
```

Comment:
If you rebase onto the latest master, you'll see this needs a `scopeExpression` giving the scopes it requires.